server/app/services/applicant/ReadOnlyApplicantProgramService.java

```diff
@@ -65,8 +65,8 @@ public interface ReadOnlyApplicantProgramService {
    */
   ImmutableList<ApplicantQuestion> getIneligibleQuestions();
 
-  /** Get the block with the given block ID */
-  Optional<Block> getBlock(String blockId);
+  /** Get the active block with the given block ID */
+  Optional<Block> getActiveBlock(String blockId);
```

Comment:
**question (blocking):** clarify what "active" means. also clarify in the javadoc for this method what will happen if the `blockId` parameter is for a block that is not active.

Reply:
added more comments in the javadoc explaining what `active` means and what will happy if the `blockId` parameter is for a block that is not active.